### PR TITLE
NPM setup and CD pipeline

### PR DESCRIPTION
**Pull request by sakari-malkki**
package.json and package-lock.json updated to use the @statisticsfinland npm organization scope
CD pipeline that publishes npm packages for dev, test and prod (latest)

### CODE REVIEW

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: PxVisualizer CD
+
+on:
+  push:
+      branches:
+        - develop
```
Comment (jsaarimaa) [inline .github/workflows/CD.yml:6]: dev and main

File: .github/workflows/CD.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        - master
+        - test
+  workflow_dispatch:
+  
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```
Comment (jsaarimaa) [inline .github/workflows/CD.yml:15]: Add node version to matrix

File: .github/workflows/CD.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      
+    - name: Set prelease version
+      run: npm run version-helper setpreversion
+      if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/test'
+
+    - name: Publish prelease version
+      run:  |
+        branchName=${GITHUB_REF##*/}
+        npm publish --tag $branchName --git-tag-version false
+      if: github.ref == 'refs/heads/develop' || 'github.ref == 'refs/heads/test'
```
Comment (jsaarimaa) [inline .github/workflows/CD.yml:55]: fix branch name: dev

File: .github/workflows/CD.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+    - name: Publish release
+      run: npm publish --tag latest --git-tag-version false
+      if: github.ref == 'refs/heads/master'
```
Comment (jsaarimaa) [inline .github/workflows/CD.yml]: Fix the branch name: main

**Review by lehtu — APPROVED** [review]

**Pull request merged**